lib/OpenQA/WebAPI/Controller/API/V1/JobGroup.pm

```diff
@@ -17,89 +17,110 @@ package OpenQA::WebAPI::Controller::API::V1::JobGroup;
 use Mojo::Base 'Mojolicious::Controller';
 use OpenQA::Schema::Result::JobGroups;
 
-sub list {
+# Helper methods
+
+sub is_parent {
+    my ($self) = @_;
+    return $self->req->url->path =~ qr/.*\/parent_groups/ ? 1 : 0;
```

Comment:
why are you casting the match to a number? returning of the return of the =~ would do.